docs/user-guide/map-views.md

```diff
@@ -0,0 +1,77 @@
+
+<img src="../img/map-views/map-views-panel.jpg" class="ms-docimage"/>
+
+To create a new simple view the user can simply move the map to the interested area, enable the desired layers in TOC to be displayed in the map and finally click on the <img src="../img/button/+++.jpg" class="ms-docbutton" /> button. The view will be created and visible in the *Views* list by clicking the <img src="../img/button/timeline-layers-list-button.jpg" class="ms-docbutton" /> button.
+
+<img src="../img/map-views/add-view.gif" class="ms-docimage"/>
+
+[MapStore](https://mapstore.geosolutionsgroup.com/mapstore/#/) allows to customize and edit the new view by opening the Edit panel through the <img src="../img/button/editing-button.jpg" class="ms-docbutton" /> button. Here the user is allowed to:
+
+* Add a text, images, videos or hyperlinks by clicking **Description** section. The *description* is visible during the [Views Navigation](map-views.md#3d-views-navigations). Take a look at the following example.
+
+<img src="../img/map-views/description.jpg" class="ms-docimage"/>
+
+* *Capture the view positions* through the <img src="../img/button/capture-view-position-button.jpg" class="ms-docbutton" /> button or change the longitude, latitude and height of the *Camera Position* and *Center Position*  (only available for the *3D* mode) by clicking on **Position** section.
+
+<img src="../img/map-views/position.jpg" class="ms-docimage"/>
+
+* Modify the duration of the animation and enable/disable the transition during the [Views Navigation](map-views.md#3d-views-navigations) by clicking on **Animation** section.
+
+<img src="../img/map-views/animation.jpg" class="ms-docimage"/>
+
+* Select a WFS or Vector layer available in the map to create a mask on all the 3D tiles visible in the map by clicking **Mask** section. The WFS or Vector layer is added to the map by the user and then used the layers features in a transparent way to create the clipping area.
+
+<img src="../img/map-views/mask-panel.jpg" class="ms-docimage"/>
+
+If the user enables the **Inverse** option, a hole is created inside the 3D Tile layers. This hole can be filled by only 3D objects (3D Tiles cannot be used to fill these holes because all 3D tiles will be masked).
+
+<img src="../img/map-views/mask-inverse.jpg" class="ms-docimage"/>
+
+If the user disables the **Inverse** option the clipping style is used for the view, then the selected WFS layer masks the 3D Tile layers outside its polygon.
+
+<img src="../img/map-views/mask.jpg" class="ms-docimage"/>
+
+* Enable the translucency of the globe in **Globe Translucency** section so that it is possible to see layers beneath the globe's surface.
+
+<img src="../img/map-views/translucency.jpg" class="ms-docimage"/>
+
+* Customize the layer present on the view by clicking **Layer Options** section. The user can simply enable or disable all the layers present on [TOC](toc.md#table-of-contents) and change the opacity of the layer.
```

Comment:
* Choose which layer should be visible when the view is active through the **Layer Options** section. In 2D mode the user can simply enable or disable all the layers present on [TOC](toc.md#table-of-contents) and change the opacity.